h2o-py/h2o/estimators/infogram.py

```diff
@@ -896,6 +896,7 @@ def plot(self, train=True, valid=False, xval=False, figsize=(10, 10), title="Inf
 
         plt = get_matplotlib_pyplot(server, raise_if_not_available=True)
         polycoll = get_polycollection(server, raise_if_not_available=True)
+        np = _get_numpy("Infogram plots")
```

Comment:
use `can_use_numpy()` instead and raise immediately with custom message or applies fallback logic if needed.